components/Wizard/formComponents/Review.js

```diff
@@ -95,6 +97,21 @@ const ociReview = (formValues) => (
 const Review = (props) => {
   const { getState } = useFormApi();
   const formValues = getState()?.values;
+  const [dependencies, setDependencies] = useState(undefined);
+  const [packages, setPackages] = useState(undefined);
+
+  useEffect(() => {
+    const fetchDependencies = async (blueprintName) => {
+      const result = await composer.depsolveBlueprint(blueprintName);
+      const numPackages =
+        result.blueprints[0].blueprint.packages.length - result.blueprints[0].blueprint.modules.length;
```

Comment:
See #1463.

However, with respect to this specific line, since we will now convert any modules to packages, modules.length should always be 0 and we can remove the subtrahend. And actually, I think I actually had intended to add them together and not subtract them!